CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Node.js Website Contributing Guide
 
-Thank you for your interest in contributing to the Node.js Website. Before you proceed, briefly go through the following:
+Thank you for being so interested in contributing to the Node.js Website. Before you proceed, briefly go through the following:
```

Comment:
i think i flagged this one before too - it reads to my ears as patronizing and should be reverted